service/src/main/java/bio/terra/workspace/common/utils/ManagementExceptionUtils.java

```diff
@@ -22,4 +25,19 @@ public class ManagementExceptionUtils {
   public static boolean isExceptionCode(ManagementException ex, String exceptionCode) {
     return StringUtils.equals(ex.getValue().getCode(), exceptionCode);
   }
+
+  public static Optional<HttpStatus> getHttpStatus(ManagementException me) {
+    try {
+      return Optional.of(HttpStatus.valueOf(me.getResponse().getStatusCode()));
+    } catch (Throwable t) {
```

Comment:
Nit: Catching throwable here seems overly broad, I don't think there is anything that can reasonably done for `OutOfMemoryError` for example.

Reply:
I am wary of NPE, I could just catch runtime